app/views/registration/chooseYourRoute.scala.html

```diff
@@ -71,7 +71,11 @@ <h2 class="heading-medium">Confirm the following statements are true</h2>
                         '_label -> "Have you applied to the Civil Service Fast Stream this year?") {
                         <div id="hasAppliedToFaststream-panel-yes" class="toggle-content">
                             <div class="panel-danger" role="alert">
-                                <p>If you want to apply for the Summer Diversity Internship Programme and have an existing Fast Stream account, you need to <a rel="external" target="_blank" href="https://www.faststream.gov.uk/summer-diversity-internship-programme/">sign in</a>.</p>
+                                <div class="text">
+                                    <p>If you want to apply for the Summer Diversity Internship Programme and have an existing Fast Stream account, you need to
+                                        <a href="@routes.SignInController.sdipPresent()">sign in</a>.
```

Comment:
I would change this 
<a href="@routes.SignInController.sdipPresent()">sign in</a>.

to 
<a href="@routes.SignInController.considerMeForSdipPresent()">sign in</a>.

to distinguish better with "normal" sdip.